boards/DCM/Src/App/states/App_InitState.c

```diff
@@ -42,6 +42,17 @@ static void InitStateRunOnEntry(struct StateMachine *const state_machine)
 
 static void InitStateRunOnTick1Hz(struct StateMachine *const state_machine)
 {
+    struct DcmWorld *world = App_SharedStateMachine_GetWorld(state_machine);
+    struct DcmCanRxInterface *can_rx = App_DcmWorld_GetCanRx(world);
+
+    // Clear inverter fault if requested by a PCAN node
+    App_SharedStates_HandleClearInvFaultsCmd(
+        App_DcmWorld_GetCanTx(world), can_rx);
```

Comment:
why do we need to do this from the init state?

Reply:
good call I'll take it out

Reply:
only clear inverter fault in fault state

Reply:
Moved `App_SharedStates_HandleClearInvFaultsCmd()` to App_FaultState and converted to static fxn `App_HandleClearInvFaultsCmd()`